backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/label/sscc/label_migros.jrxml

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.21.2.final using JasperReports Library version 6.21.2-8434a0bd7c3bbc37cbf916f2968d35e4b165821a  -->
```

Comment:
Why are we adding this here?
Is a customer label

Reply:
<img width="574" alt="image" src="https://github.com/user-attachments/assets/cafbf175-1813-49b8-8321-eb75d423434e" />